networking/using-rfhe.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="using-rfhe"]
+= Monitoring Redfish hardware events
+include::modules/common-attributes.adoc[]
+:context: using-rfhe
+
+toc::[]
+
+:FeatureName: Redfish hardware events
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="about-using-redfish-hardware-events"]
+== About Redfish hardware events
+
+{product-title} enables your monitoring services to consume cloud-native events from Redfish hardware event-capable equipment. The events are published on a node and transmitted on an advanced message queue to your subscribing applications.
+
+Redfish hardware events provide the basis for a standard network monitoring services that is cloud friendly and secure. The standard is developed under the guidance of the DMTF (formerly referred to as the Distributed Management Task Force). Redfish provides an industry-standard protocol with a RESTful interface. The protocol can be used for the management of distributed, converged or software-defined resources and infrastructure. For more information, see the link:https://www.dmtf.org/standards/redfish[Redfish home page].
+
+[NOTE]
+====
+Redfish hardware events work with Redfish-capable devices on a single-node cluster provisioned only on bare-metal infrastructure.
+====
+
+The hardware-related events published through Redfish to monitoring applications may include:
+
+* breaches of temperature limits
+* server status
+* router status
+* fan status
+
+Application developers can the Redfish hardware events API to get log updates on things that have or could go wrong in a distributed system.
+
+A typical use case is the monitoring of hardware faults in Edge devices such as distributed units (DUs) and  virtual distributed units (vDUs) and other forms of advanced communication devices.
```

Comment:
vDU or DU?
If it's a physical network function, OpenShift is not in the picture.
I suggest we skip all references to vDU and just focus an SNO, no matter what it's used for

Reply:
Removed references to vDU.